proxy/http/HttpConfig.cc

```diff
@@ -1255,6 +1257,7 @@ HttpConfig::reconfigure()
   params->oride.parent_connect_attempts       = m_master.oride.parent_connect_attempts;
   params->per_parent_connect_attempts         = m_master.per_parent_connect_attempts;
   params->parent_connect_timeout              = m_master.parent_connect_timeout;
+  params->parent_failures_update_hostdb       = m_master.parent_failures_update_hostdb;
```

Comment:
Can we make this configuration overridable instead? Or are we positive that this is something we always want completely on, or completely off? In general, new configurations ought to be overridable unless it's impossible to do so (i.e. configs outside of the HttpSM are difficult to make overridable).